README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
+  - [pip-installed](https://deephaven.io/core/docs/tutorials/quickstart-pip/)
+- Groovy
+  - [Run from Docker](https://deephaven.io/core/groovy/docs/tutorials/quickstart/)
+- [Python client](https://pypi.org/project/pydeephaven/)
+- [Java client](https://deephaven.io/core/docs/how-to-guides/java-client/)
+- [JS client](https://deephaven.io/core/docs/reference/js-api/documentation/)
+- [Go client](https://pkg.go.dev/github.com/deephaven/deephaven-core/go)
+- [R client](https://github.com/deephaven/deephaven-core/blob/main/R/rdeephaven/README.md)
 
-You will need to `cd` into the deployment directory to launch or interact with the deployment.
+Deephaven's client APIs use [gRPC](https://grpc.io/), [protobuf](https://github.com/deephaven/deephaven-core/tree/main/proto/proto-backplane-grpc/src/main/proto/deephaven/proto), [Apache Arrow Flight](https://arrow.apache.org/docs/format/Flight.html), and [Barrage](https://github.com/deephaven/barrage) to handle ticking data. Users who wish to build their own client APIs can use these tools to do so.
 
-```bash
-mkdir deephaven-deployment
-cd deephaven-deployment
-```
-
-> :warning: **Commands in the following sections for interacting with a deployment must be run from the deployment directory.**
-
-### Launch: Python
-
-Run the following commands to launch Deephaven for Python server applications. 
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/base/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with NLTK
-
-Run the following commands to launch Deephaven for Python server applications with the [NLTK](https://nltk.org/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/NLTK/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with PyTorch
+## Run the Deephaven server
 
-Run the following commands to launch Deephaven for Python server applications with the [PyTorch](https://pytorch.org/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/PyTorch/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+### From Docker
 
-### Launch: Python with SciKit-Learn
+This is the easiest way to get started with Deephaven. For full instructions, see our [quickstart for Docker](https://deephaven.io/core/docs/tutorials/quickstart/).
 
-Run the following commands to launch Deephaven for Python server applications with the [SciKit-Learn](https://scikit-learn.org/stable/) module pre-installed.
+TL;DR:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+# Python
+docker run --rm --name deephaven -p 10000:10000 ghcr.io/deephaven/server:latest
```

Comment:
When I scanned the rendered document, I had to reread this section 4-5 times to figure out that these were instructions for groovy and python.  The doc should be structured so that this is super obvious e.g. `Python` and `Groovy` headings.

Also, it isn't obvious that the "TL;DR"  phrase is adding anything to the document.